descriptions/ab+/en_us.lua

```diff
@@ -819,38 +819,38 @@ EID.descriptions["en_us"].sacrifice={
 	{"1", "", "50% chance for 1 Coin#100% chance for 1 Bomb, when on B1 and killed the Boss under 1 minute"},
 	{"2", "", "50% chance for 1 Coin#100% chance for 1 Bomb, when on B1 and killed the Boss under 1 minute"},
 	{"3", "", "67% chance for increased {{AngelRoom}}Angel Room chance"},
-	{"4", "", "50% chance for 1 random Chest"},
-	{"5", "", "33% chance for 3 Coin#67% chance for increased Angel{{AngelRoom}} Room chance"},
-	{"6", "", "33% chance for teleport to {{AngelRoom}}Angel Room#Teleports to {{DevilRoom}}Devil Room when its door was spawned on this floor already#67% chance for random Chest"},
+	{"4", "", "50% chance for 1 random chest"},
+	{"5", "", "33% chance for 3 Coins#67% chance for increased {{AngelRoom}}Angel Room chance"},
+	{"6", "", "33% chance for a teleport to the {{AngelRoom}}Angel Room#Teleports to the {{DevilRoom}}Devil Room instead if its door was spawned on the current floor#67% chance for random chest"},
 	{"7", "", "33% chance for 1 random {{AngelRoom}}Angel Room item#67% chance for 1 Soul Heart"},
 	{"8", "", "100% chance for 7 troll bombs"},
-	{"9", "", "100% chance to spawn \"Uriel\" boss"},
+	{"9", "", "100% chance to spawn \"Uriel\""},
 	{"10", "", "50% chance for 7 Soul Hearts#50% chance for 30 coins"},
-	{"11", "", "100% chance to spawn \"Gabriel\" boss"},
-	{"12", "", "50% chance to teleport to \"Dark Room\" floor"},
+	{"11", "", "100% chance to spawn \"Gabriel\""},
+	{"12", "", "50% chance to teleport to the \"Dark Room\""},
 }
 
 ---------- Dice Room ----------
-EID.descriptions["en_us"].diceHeader = "[Dice effect]"
+EID.descriptions["en_us"].diceHeader = "[Dice effects]"
 
 EID.descriptions["en_us"].dice={
-	{"1", "", "Rerolls all your collectibles into items from random item pools"},
+	{"1", "", "Rerolls all your passive items into a random item from the room pool it was acquired from"},
 	{"2", "", "Rerolls all pickups in the room"},
-	{"3", "", "Rerolls all pickups on the entire floor"},
-	{"4", "", "Rerolls all pedestal items on the floor#Doesn't affect Devil deals"},
-	{"5", "", "Reroll and restart the current floor"},
-	{"6", "", "Reroll all of your items and all the pickups & items on the entire floor"}
+	{"3", "", "Rerolls all pickups and trinkets on the entire floor#Does not affect the {{DevilRoom}}Devil or {{AngelRoom}}Angel Room"},
+	{"4", "", "Rerolls all pedestal items on the floor#Does not affect the {{DevilRoom}}Devil or {{AngelRoom}}Angel Room, {{Shop}}Shop or {{Library}}Library"},
```

Comment:
I think those will generate icons for the shop and library, but not 100% sure.

Reply:
Yep thats correct.
Fyi: here is a list of most of the icons you can use: 
https://github.com/wofsauge/External-Item-Descriptions/wiki/Markup

Reply:
Oh thanks, forgot that was there.